AppServer/google/appengine/tools/devappserver2/python/sandbox.py

```diff
@@ -819,8 +823,7 @@ class BuiltinImportHook(object):
   """
 
   def find_module(self, fullname, unused_path=None):
-    if (fullname in sys.builtin_module_names and
-        fullname not in _WHITE_LIST_C_MODULES):
+    if fullname in sys.builtin_module_names and fullname not in _WHITE_LIST_C_MODULES:
```

Comment:
We usually try to keep lines under 80 characters as recommended in [PEP 8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).